intelmq/bots/experts/cut_from_string/expert.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+"""
+Cut from string
+"""
+from intelmq.lib.bot import Bot
+
+
+class CutFromStringExpertBot(Bot):
+    string_from_start: int = 1  # 1 - from start, 0 - from end
+    string_for_cut: str = 'www.'
+    field_for_cut: str = 'source.fqdn'
+
+    def init(self):
+        pass
+
+    def process(self):
+        event = self.receive_message()
+
+        if self.field_for_cut in event:
+            field_string = event[self.field_for_cut]
+            if self.string_from_start == 1 and field_string.startswith(self.string_for_cut):
+                field_string = field_string[len(self.string_for_cut):]
+                event.change(self.field_for_cut, field_string)
+
+            if self.string_from_start == 0 and field_string.endswith(self.string_for_cut):
+                field_string = field_string[:-len(self.string_for_cut)]
+                event.change(self.field_for_cut, field_string)
+
+        self.send_message(event)
+        self.acknowledge_message()
+
```

Comment:
PEP616 lists example code for the `removeprefix` and `removesuffix` String methods- maybe we can use those here and add a comment to drop them once we only support Python versions that implement those methods:
```
def removeprefix(self: str, prefix: str, /) -> str:
    if self.startswith(prefix):
        return self[len(prefix):]
    else:
        return self[:]

def removesuffix(self: str, suffix: str, /) -> str:
    # suffix='' should not call self[:-0].
    if suffix and self.endswith(suffix):
        return self[:-len(suffix)]
    else:
        return self[:]
```

then the code would look like this:
```
if self.removeprefix:
    event.change(self.field, self.removeprefix(self.field, self.affix)
else:
    event.change(self.field, self.removesuffix(self.field, self.affix)
```